doc/Overview/Mvux/WIP_Overview.md

```diff
@@ -0,0 +1,179 @@
+﻿---
+uid: Overview.Mvux.Overview
+---
+
+# MVUX Overview
+
+MVUX is a variation of the MVU design pattern that will also feel familiar to developers who have previously worked with MVVM.
+
+Considering a simple application scenario helps to understand what it does and the benefits gained from using it, so let's do that.
+
+Our application will show the current temperature based on an external source.
+
+That should seem simple enough. All it needs to do is put a number on the screen.
+Although this seems like an easy problem, as is often the case, there are more details to consider than may be immediately apparent.
+
+- What if the external data isn't immediately available when starting the app?
+- How to show that data is being initially loaded? Or updated?
+- What if no data is available?
+- What if there's an error in obtaining or processing the data?
+- How to keep the app responsive while requesting or updating the UI?
+- How to keep track of the selected item or items?
```

Comment:
It's not obvious how this relates to the app described above. What is there to select?